debezium-core/src/main/java/io/debezium/metrics/Metrics.java

```diff
@@ -29,6 +30,7 @@
 
     private final ObjectName name;
     private volatile boolean registered = false;
+    private static final Logger logger = LoggerFactory.getLogger(Metrics.class);
```

Comment:
s/logger/LOGGER/g; and this should go before the non-static fields.

Reply:
This might be something worth enforcing at the coding standard level 😉

Reply:
Yes, indeed. @Naros, are you up to looking into this perhaps:

- enforce static members come first
- enforce static member names are UPPER_CASED

?

Not sure though how large the change set this creates would be.

Reply:
Logged https://issues.redhat.com/browse/DBZ-4551 for this.